### DISCUSSION TITLE
how to get started

### DISCUSSION THREAD

**Discussion by nguyenduy1234** [pkgxdev/pkgx#942] [Help]
go

**Comment by dangaoron** [comment]
you can start by being active on discord, engaging in there while waiting for the testnet

**Comment by codewithneeraj** [comment]
For Now you can Submit your Email here> https://tea.xyz/early-rewards

You will receive an email from tea where you will have to reply ok to that mail.

NOTE: Testnet will begin from 21st February where you can perform different Quests to earn Points.

I hope It's helpful.
